### ISSUE TITLE
Will the code for training value noise model and hole noise model be released？

### ISSUE THREAD

**Issue by lu-ming-lei** [ByteDance-Seed/manip-as-in-sim-suite#5]
Thanks for your great work! 
Will you release the training code and detailed training strategies for the hole noise model and value noise model?

**Comment by Ericonaldo** [comment]
Hi! Thanks for liking this work!

Currently, we do not have any open-sourcing plan for training code. But I want to highlight the hole noise is just a binary classification model with Dino-V2, and the value noise model is just a finetuned DAV2 model (with MoGe-like training objective). The key is the guided filter to align the metric of the predicted depth to the GT.
